Cesium.externs.js

```diff
@@ -3416,3 +3444,37 @@ Cesium.EntityView.prototype.update = function(currentTime, bs) {};
  * @constructor
  */
 Cesium.CallbackProperty = function(cb, constant) {};
+
+/**
+ * @param {Cesium.BoundingSphere} occluderBoundingSphere
+ * @param {Cesium.Cartesian3} cameraPosition
+ * @constructor
+ */
+Cesium.Occluder = function(occluderBoundingSphere, cameraPosition) {};
+
+/**
+ * @param {Cesium.Cartesian3} occludee
+ */
+Cesium.Occluder.prototype.isPointVisible  = function(occludee) {};
+
+/**
+ * @enum {number}
+ */
+Cesium.Intersect = {
+  OUTSIDE: -1,
+  INTERSECTING: 0,
+  INSIDE: 1
+};
+
+
+/**
+ * @param {Array.<Cesium.Cartesian4>} planes
+ * @constructor
+ */
+Cesium.CullingVolume = function(planes) {};
+
+/**
+ * @param {Object} boundingVolume
```

Comment:
The type should be `Cesium.BoundingSphere`.

Reply:
I took the api description from the cesium API: https://cesiumjs.org/Cesium/Build/Documentation/CullingVolume.html
So computeVisibility works with every object which implements intersectsPlane().

Reply:
OK, then fine with it since it would be too cumbersome to introduce interfaces.